src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java

```diff
@@ -43,7 +41,6 @@ public CaseDocument bundle(
         String bundleFilename,
         CaseData caseData
     ) {
-        CaseDocument caseDocument = null;
         CaseDetails payload =
```

Comment:
Minor: While we're at it, can I ask that this call be put on a single line? It's seems really overkill that they split it with each argument on a different line...

Reply:
Done